components/Sidebar.tsx

```diff
@@ -178,16 +180,19 @@ export default function Sidebar({ children }: { children: React.ReactNode }) {
           />
 
           <div className="flex flex-1 gap-x-4 self-stretch items-center lg:gap-x-6 justify-end">
-            <div className="flex-shrink-0">
-              <Link
-                href="/documents/new"
-                type="button"
-                className="relative inline-flex items-center gap-x-1.5 rounded-md bg-indigo-500 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-500"
-              >
-                <PlusIcon className="-ml-0.5 h-5 w-5" aria-hidden="true" />
-                New Link
-              </Link>
-            </div>
+            {!isDocumentsRoute ? (
```

Comment:
You need to add a check in to show the "new link" button if there are documents available. 

Right now it only checks for the route and assumes `documents.length == 0`